themes/bootstrap3/templates/search/reservessearch.phtml

```diff
@@ -22,12 +22,18 @@
     <div class="resulthead">
       <div class="pull-left flip">
         <? if (($recordTotal = $this->results->getResultTotal()) > 0): ?>
-          <?=$this->transEsc("Showing")?>
-          <strong><?=$this->localizedNumber($this->results->getStartRecord())?></strong> - <strong><?=$this->localizedNumber($this->results->getEndRecord())?></strong>
-          <?=$this->transEsc('of')?> <strong><?=$this->localizedNumber($recordTotal)?></strong>
-          <?=$this->transEsc('for search')?>: <strong>'<?=$this->escapeHtml($reservesLookfor)?>'</strong>,
+          <?
+            $transParams = [
+              '%%start%%' => $this->localizedNumber($this->results->getStartRecord()),
+              '%%end%%'   => $this->localizedNumber($this->results->getEndRecord()),
+              '%%total%%' => $this->localizedNumber($recordTotal),
+              '%%lookfor%%' => $this->escapeHtml($reservesLookfor)
+            ];
+          ?>
+          <?=$this->translate('showing_results_of_for_html', $transParams); ?>
         <? endif; ?>
         <? if ($qtime = $this->results->getQuerySpeed()): ?>
+          <? if ($recordTotal > 0) ?>, <? endif ?>
```

Comment:
This looks like another manifestation of the "space before comma" problem you fixed elsewhere. Might want to check and see if other templates are affected. Let me know if you need help setting up a test instance with reserves search active.